app.go

```diff
@@ -887,7 +886,7 @@ func (app *App) start(ctx context.Context) error {
 	// Attempt to start cleanly.
 	if err := app.lifecycle.Start(ctx); err != nil {
 		// Start failed, rolling back.
-		app.log.LogEvent(&fxevent.Rollback{StartErr: err})
+		app.log.LogEvent(&fxevent.RollingBack{StartErr: err})
 		if stopErr := app.lifecycle.Stop(ctx); stopErr != nil {
 			app.log.LogEvent(&fxevent.RollbackError{Err: stopErr})
```

Comment:
Ugh, yea, I struggled with this one as it would have two error fields which are easy to confuse if combined. Let me update this by either combining and making error fields super distinct or leaving them separate.